src/patternfly/components/Alert/alert-action.hbs

```diff
@@ -1,6 +1,6 @@
-<div class="pf-c-alert__action{{#if alert-action--modifier}} {{alert-action--modifier}}{{/if}}"
+<a class="pf-c-alert__action{{#if alert-action--modifier}} {{alert-action--modifier}}{{/if}}"
```

Comment:
Oh, sorry if I was unclear. `.pf-c-alert__action` should be a div. It's the "action" element of the alert component that holds the action things (i.e., buttons)

Reply:
In @michael-coker's original comment:

> In this case, "Reload server" seems like an action, and a <button> would be more fitting. I would be inclined to move this to the .pf-c-alert__action element as a <button>. If you look at this link in the PF3 toast notification, it has .pull-right, grouping it to the right edge of the toast notification, separating it from the notification text.

I interpreted his suggestion to mean that the action "Reload Server" should be pulled out of `.pf-c-alert__body` and moved to `.pf-c-alert__action` so that it displays on the right side, with the X close button. 

I definitely think we should go ahead with this change, even though the design question of actions on toasts is still being worked out. We shouldn't include any actionable elements inside `role="alert"` given that those elements are not immediately accessible to screen reader users.

Reply:
thanks for clarifying, changed to div

Reply:
@jgiardino thanks for clarifying, I was a little confused. Reload Server is now a button with a modifier of "pf-m-link" inside of the alert__action